sdks/how-to/how-to-manage-notifications.md

```diff
@@ -4,3 +4,181 @@ description: Learn how to manage notifications
 tags:
 - Notification
 ---
+
+# How to Manage Notifications
+
+## What Is a Notification?
+
+A Notification represent a request from a [Data Owner](/sdks/glossary#data-owner) to a Healthcare Professional to perform 
+an operation.  
+As for now, there are three types of Notifications:
+
+* **KEY_PAIR_UPDATE**: when a Patient loses their private key and gets a new one, they can send this type of Notification to a Healthcare Professional to ask them to share their data with them again.
+* **NEW_USER_OWN_DATA_ACCESS**: when a Patient logs in for the first time, they can send this type of Notification to a Healthcare Professional to aks for access to their own data.
+* **OTHER**: all the other use cases.
+
+:::note
+
+To perform the following operations, we suppose you have at least a Patient and a Healthcare Professional in your database.
+
+:::
+
+## Creating a Notification
+
+In the following example, a Patient creates a Notification for a Healthcare Professional communicating that they have a new
+ key and need access to their data.
+
+<!-- file://code-samples/how-to/manage-notifications/index.mts snippet:create a notification as patient-->
+```typescript
+const accessNotification = await patientApi.notificationApi.createOrModifyNotification(
+  new Notification({
+    type: NotificationTypeEnum.KEY_PAIR_UPDATE,
+  }),
+  hcp.id,
+)
+```
+
+:::note
+
+The default status of a Notification is `pending`
+
+:::
+
+## Retrieving a Notification
+
+### Retrieving a Notification Using its Id
+
+In the following example, a Patient creates a Notification for a Healthcare Professional and then retrieves it using its
+ id.
+
+<!-- file://code-samples/how-to/manage-notifications/index.mts snippet:creates a notification, then retrieves it-->
+```typescript
+const createdNotification = await patientApi.notificationApi.createOrModifyNotification(
+  new Notification({
+    type: NotificationTypeEnum.OTHER,
+  }),
+  hcp.id,
+)
+
+const retrievedNotification = await patientApi.notificationApi.getNotification(
+  createdNotification.id,
+)
+```
+
+### Retrieving Notifications Using Complex Criteria
+
+If you want to retrieve a set of Notifications that satisfy complex criteria, you can use a Filter.  
+In this example, a Healthcare Professional filters all their Notifications that were created after a certain date.
+
+<!-- file://code-samples/how-to/manage-notifications/index.mts snippet:creates after date filter-->
+```typescript
+const startTimestamp = new Date(2022, 8, 27).getTime()
+
+const afterDateFilter = await new NotificationFilter()
+  .forDataOwner(user.healthcarePartyId)
+  .afterDateFilter(startTimestamp)
+  .build()
+```
+
+:::note
+
+You can learn more about filters in the how to
+
+:::
```

Comment:
```suggestion
```
There is no general filter how to, at least currently

Reply:
Is not the one Lisa is writing?